astro/src/content/blog/advanced-registration-form.mdx

```diff
@@ -95,8 +95,12 @@ Fields can appear in any order on the form; whatever makes the most sense for yo
 The only requirements for a registration form are:
 
 * You must have at least one step.
-* You must have either an email or a username field in one of your steps.
-* You must have a password field in one of your steps.
+* You must have either an email, phone number, or a username field in one of your steps.
+* You must have a password field in one of your steps. (see note below)
+
+<Aside type="note" title="Passwordless Registration">
+  Because forms can be used with multiple tenants, they must include a password field. If a tenant has <InlineField>tenant.passwordEnabled</InlineField> set to `false`, then the password field is not displayed to users during registration.
```

Comment:
```suggestion
  Because forms can be used with multiple tenants, they must include a password field, even if a tenant has <InlineField>tenant.passwordEnabled</InlineField> set to `false`. For tenants that set the field to `false`, the password field is not displayed to users during registration.
```

Reply:
My suggestion may be worse, idk, but I was attempting to tie the consequences together there.